### ISSUE TITLE
Swiching language creates long URL

### ISSUE THREAD

**Issue by pmeulen** [OpenConext/OpenConext-teams#15]
Result of switch between EN and NL:
```
https://teams.surfconext.nl/teams/?lang=EN&
https://teams.surfconext.nl/teams/?lang=EN&lang=NL&
https://teams.surfconext.nl/teams/?lang=EN&lang=NL&lang=EN&
```
This causes (some) automated scanners to get stuck in an endless loop.

**Comment by oharsta** [comment]
Fixed on the develop branch and deployed on the the test environment. Candidate release for acceptance

**Comment by thijskh** [comment]
Fixed in 909733dfda54069c18b520cd07200a381faf81c8
